File: day5/myJavaFiles/JvmOverloadsWithThreeDefaults.java
package day5.myJavaFiles;

import day5.myKotlinFiles.JvmOverloadsWithThreeDefaultsKt;

public class JvmOverloadsWithThreeDefaults {
    public static void main(String[] args){
        JvmOverloadsWithThreeDefaultsKt.printDetails();
        JvmOverloadsWithThreeDefaultsKt.printDetails("Rushi");
        JvmOverloadsWithThreeDefaultsKt.printDetails("Rushi",18);
    }
}

File: day5/myKotlinFiles/JvmOverloadsWithThreeDefaults.kt
package day5.myKotlinFiles

fun main(args: Array<String>){
    printDetails()
    printDetails("Rushi")
    printDetails("Rushi",18)
}
@JvmOverloads
fun printDetails(name:String="Unknown",age:Int=0,country:String="India"){
    println("Name: $name, Age: $age, Country: $country")
}